icu4j/main/classes/core/src/com/ibm/icu/util/NoUnit.java

```diff
@@ -7,47 +7,43 @@
 /**
  * Dimensionless unit for percent and permille.
  * @see NumberFormatter
- * @draft ICU 60
+ * @draft ICU 68
  * @provisional This API might change or be removed in a future release.
  */
-public class NoUnit extends MeasureUnit {
-    private static final long serialVersionUID = 2467174286237024095L;
-
+public final class NoUnit {
```

Comment:
I believe that this class is still trivially instantiable. (You could experiment with that.)
If so, please add a private, empty default constructor.

Reply:
Done.